nim_status/client.nim

```diff
@@ -224,6 +230,36 @@ proc storeImportedWalletAccount(self: StatusObject, privateKey: SkSecretKey,
   except Exception as e:
     return WalletAccountResult.err e.msg
 
+proc addCustomToken*(self: StatusObject, address, name, symbol, color, decimals: string): AddCustomTokenResult =
+  if not self.isLoggedIn:
+    return AddCustomTokenResult.err "Not logged in. You must be logged in to " &
+      "add a new custom token."
+
+  var uintDecimals: uint
+  if decimals != "":
+    uintDecimals = parseUInt(decimals)
+  # TODO
+  let token = Token(address: cast[Address](address), name: name, symbol: symbol, color: color, decimals: uintDecimals)
+  try:
+    self.userDb.addCustomToken(token)
+  except Exception as e:
+    return AddCustomTokenResult.err "Error adding a custom token: " & e.msg
+
+  AddCustomTokenResult.ok(token)
+
+proc deleteCustomToken*(self: StatusObject, address: string): DeleteCustomTokenResult =
```

Comment:
Here too, I think it would be better to accept a type of `Address`:
```suggestion
proc deleteCustomToken*(self: StatusObject, address: Address): DeleteCustomTokenResult =
```

Reply:
Done.